app/Http/Controllers/User/NotificationController.php

```diff
@@ -28,23 +28,26 @@ public function getNotifications($limit = -1)
         $where = [
             'receive_id' => Auth::id(),
         ];
-
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+        
+        return $data = $this->notification->getNotifications($with, $where);
     }
 
     public function getAllNotifications()
     {
-        $notifications = $this->getNotifications();
+        $notifications = $this->notification->paginate(
+            $this->getNotifications(), 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 38</br>Whitespace found at end of line